systems/plants/RigidBody.h

```diff
@@ -45,7 +45,8 @@ class DLLEXPORT_RBM RigidBody {
   static const std::set<int> defaultRobotNumSet;
 // note: it's very ugly, but parent,dofnum,and pitch also exist currently (independently) at the rigidbodymanipulator level to represent the featherstone structure.  this version is for the kinematics.
   int parent;
-  int dofnum;
+  int dofnum; // interpreted as start of position_num from Matlab
```

Comment:
Yes, but my reason for leaving it like this for now is to not cause merge conflicts all over the place. Maybe it'll be easier to do a separate pull request where we rename all of these things right after the DRC meeting so that people won't be affected as much.